addons/l10n_tr_nilvera_einvoice/models/account_move_send.py

```diff
@@ -24,6 +25,16 @@ def _get_all_extra_edis(self) -> dict:
     # -------------------------------------------------------------------------
 
     def _get_alerts(self, moves, moves_data):
+        def _is_valid_nilvera_name(move):
+            _, parts = move._get_sequence_format_param(move.name)
+
+            return (
+                parts['year'] != 0
+                and parts['year_length'] == 4
+                and parts['seq'] != 0
+                and re.match(r'^[A-Za-z0-9]{3}[^A-Za-z0-9]?$', parts['prefix1'])
```

Comment:
is teh regex necessary? 
Can't you just check isalnum and len==3? 
Wouldn't it be cleaner?